src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Errors/UserStringBuilder.cs

```diff
@@ -656,21 +656,20 @@ private void ErrAppendType(CType pType, SubstContext pctx, bool fArgs)
                             // Add [] with (rank-1) commas inside
                             ErrAppendChar('[');
 
-#if ! CSEE
```

Comment:
In that case, why the result would be different?

---
In reply to: [109752192](https://github.com/dotnet/corefx/pull/17810#discussion_r109752192) [](ancestors = 109752192,109408159)